tests/functional/splinter_test.c

```diff
@@ -950,6 +950,11 @@ splinter_perf_inserts(platform_heap_id             hid,
 
    uint64 start_time = platform_get_timestamp();
 
+   bool verbose_progress = show_verbose_test_progress(test_cfg->test_exec_cfg);
+   if (verbose_progress) {
```

Comment:
One illustrative usage ... to prove that this new config option is working right.